pygin/pygin/src/helpers.hpp

```diff
@@ -45,4 +46,30 @@ namespace helpers
 			return Result;
 		}
 	}
+
+	class py_string_storage
+	{
+	public:
+		py_string_storage() {}
```

Comment:
`= default`?

Reply:
I think this is an evolutionary artifact, I'll delete it altogether.

Reply:
Fixed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.